cypress/integration/segments/segments.spec.ts

```diff
@@ -9,7 +9,7 @@ const disableActiveSplashScreens = () => {
     cy.visit(`/splash/operators`);
 };
 
-describe('segments', () => {
+describe('project-access', () => {
```

Comment:
I think this was a mistake that should be reverted 🙂

Reply:
Thanks. Typo.